application/models/Campaign_model.php

```diff
@@ -114,6 +114,10 @@ public function prioritize_crawl($offices, $milestone)
                 if ($crawl['crawl_status'] == 'in_progress' && empty($prioritize[$crawl['office_id']])) {
                     $de_prioritize[$crawl['office_id']] = true;
                 }
+                // deprioritize if long_running is true
```

Comment:
Good that we're doing this even though it sounds like we'll be running the long-running crawls in a separate job without a timeout.

Reply:
Yep, good point. I wan these to be at the bottom of the `omb-monitored` "queue